### PR TITLE
CFE-2997 Fix cf-key help output to indicate ability to delete by key digest

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @olehermanse can review this?

File: cf-key/cf-key.c [hunk 1]
```diff
@@ -100,7 +100,7 @@ static const char *const HINTS[] =
     "Specify an alternative output file than the default.",
     "Specify a RSA key size in bits, the default value is 2048.",
     "Show lastseen hostnames and IP addresses",
-    "Remove keys for specified hostname/IP",
+    "Remove keys for specified hostname/IP/MD5/SHA (cf-key -r SHA=12345, cf-key -r MD5=12345, cf-key -s host001, cf-key -s 203.0.113.1)",
```
Comment (olehermanse) [inline cf-key/cf-key.c]: The last 2 examples should also be `-r`?

**Comment by olehermanse** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2084/)

https://ci.cfengine.com/job/pr-pipeline/2084/

**Pull request merged**